rule/src/main/java/htw/kbe/maumau/rule/service/RulesServiceImpl.java

```diff
@@ -1,32 +1,61 @@
 package htw.kbe.maumau.rule.service;
 
 import htw.kbe.maumau.card.domain.Card;
+import htw.kbe.maumau.card.domain.Label;
 import htw.kbe.maumau.card.domain.Suit;
-import htw.kbe.maumau.rule.domain.Rule;
 import htw.kbe.maumau.player.domain.Player;
 
 public class RulesServiceImpl implements RulesService {
-    public boolean isJoker(Card card, Rule rule) {
-        return false;
+
+    @Override
+    public boolean isCardValid(Card userCard, Card topCard) {
+
+        if (topCard.getLabel().equals(Label.JACK)) {
+            if (topCard.getLabel().equals(userCard.getLabel())) {
+                return false;
+            } else {
+                return true;
+            }
```

Comment:
```suggestion
        return topCard.getLabel().equals(userCard.getLabel()
```

Gibt dann entweder true oder false zurück, nur kürzer.

Reply:
an sich korrekt aber im ersten schritt prüfe ich ob jemand versucht bube auf bube zu legen und danach prüfe ich ob die labels gleich sind oder die suits, dadurch brauche ich ja zweimal return oder verstehe ich dich gerade falsch?